### PR TITLE
DATASCI-510 create documentation for transaction and customer files

### PR DESCRIPTION
**Pull request by vicki-pt**

### CODE REVIEW

File: file-formats/customer-file/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Customer Email Hash File #
+
+This document specifies the requirements for customer email hash files.
+
+### File Specifications ###
+
+- Must be a comma-separated value (CSV) file - columns delimited by commas (,)
+- Rows separated by newline (\n) characters
+- Fields encapsulated with double quotation (") character
```
Comment (davidwarshaw) [inline file-formats/customer-file/README.md]: Should this be `All non-numeric fields...`?

File: file-formats/customer-file/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Customer Email Hash File #
+
+This document specifies the requirements for customer email hash files.
+
+### File Specifications ###
+
+- Must be a comma-separated value (CSV) file - columns delimited by commas (,)
```
Comment (davidwarshaw) [inline file-formats/customer-file/README.md]: Let's use `fields` throughout, vs `columns`.

**Review by prasadsilva — APPROVED** [review]
:shipit: if @davidwarshaw approves

**Review by davidwarshaw — APPROVED** [review]
:shipit:

**Pull request merged**